src/lib/API/api.js

```diff
@@ -680,6 +682,71 @@ export class APIService {
 
     return txs
   }
+
+  async getAltClaim(account, chainId, lastBlock) {
+    if (!lastBlock) {
+      return
+    }
+    const explorer = Config.ethereum[chainId]?.explorerAPI
+    const sender32 = ethers.utils.hexZeroPad(account, 32)
+    const claimHash = '0x89ed24731df6b066e4c5186901fffdba18cd9a10f07494aff900bdee260d1304'
+
+    const now = moment().utc()
+    let today12UTC = moment().utc().set({ hour: 12, minute: 0, second: 0, millisecond: 0 })
+
+    if (now.isBefore(today12UTC)) {
+      today12UTC = today12UTC.subtract(1, 'day')
+    }
+
+    const timestamp12UTC = today12UTC.unix()
+
+    for (;;) {
+      const apis = shuffle(explorer.split(',')).map(baseURL => async () => {
+        const paramsBlockNo = {
+          module: 'block',
+          action: 'getblocknobytime',
+          timestamp: timestamp12UTC,
+          closest: 'before',
+        }
+
+        const blockNoOptions = { baseURL, params: paramsBlockNo }
+        const { result: blockNoStart } = await this.sharedClient.get('/api', blockNoOptions)
+
+        // return []
+        if (!blockNoStart) {
+          return false
+        }
+
+        const params = {
+          module: 'logs',
+          action: 'getLogs',
+          page: '1',
+          offset: '31',
```

Comment:
why 31?

Reply:
I actually don't know. I assume I thought for 31 days, but even that does not make any sense in this context.

Reply:
changed to 10k which should be default as well